questions/qa-personal-names.fr.html

```diff
@@ -110,12 +109,12 @@ <h3 id="multiplefamily"><a href="#multiplefamily">Noms de famille multiples</a><
   <section>
     <h3 id="variants"><a href="#variants">Déclinaison de formes</a></h3>
     <p>Nous avons vu plus haut que le patronyme en Islande se termine en -son or -dóttir, selon que l'enfant est un garçon ou une fille. Les Russes utilisent le patronyme comme nom du milieu mais ils utilisent aussi le nom de famille, dans l'ordre prénom-patronyme-nomDeFamille. Les terminaisons du patronyme et du nom de famille indiqueront s'il s'agit d'un garçon ou d'une fille. Par exemple, la femme de <b class="highlight">Борис Николаевич Ельцин</b> (Boris Nikolayevich Yeltsine) est <b class="highlight">Наина Иосифовна Ельцина</b> (Naina Iosifovna Yeltsina) – notez qu'en russe les noms du mari se terminent avec une consonne, alors que les noms de la femme (y compris le patronyme qu'elle tient de son père) se terminent en ‘<span class="qchar">a</span>’.</p>
-  <p>By the way, a slightly less formal way of writing Russian names follows the order familyName-givenName-patronymic, such as <b class="highlight">Ельцина Наина Иосифовна</b>.</p>
+    <p>À propos, une manière un peu moins formelle d’écrire des noms russes est de suivre l’ordre nomDeFamille-prénom-patronyme, comme <b class="highlight">Ельцина Наина Иосифовна</b> (Yeltsina Naina Iosifovna)</p>
 </section>
   <section>
     <h3 id="middleinitial"><a href="#middleinitial">Initiales au milieu</a></h3>
     <p>Les américains écrivent souvent leur nom avec une initiale au milieu, par exemple <b class="highlight">John Q. Public</b>. Les formulaires conçus aux États-Unis considèrent fréquemment cette pratique comme normale, alors qu'au Royaume-Uni, où les gens ont souvent un ou plusieurs seconds prénoms, elle est vue comme une approche très américaine. Dans d’autres pays, les personnes avec plus de deux noms et qui ne les écrivent pas habituellement en initiales, peuvent ne pas savoir comment remplir ce type de formulaires. De plus il faut bien comprendre que de nombreuses personnes qui utilisent des initiales les mettent au début de leur nom.</p>
- <p>Filipinos also write their name with a middle initial, but it represents the mother's  name before marriage rather than a given name. For example, in <b class="highlight">Maria J. Go</b>, the initial repesents Jimenez, the previous family name of Maria's mother. (In fact, an initial may represent more than one name: 'D' may stand for 'Dela Cruz' when the name is written in full.)</p>
+    <p>Les Philippins écrivent également leur nom avec une initiale au milieu, mais celle‑ci représente le nom de naissance de la mère. Par exemple, pour <b class="highlight">Maria J. Go</b>, l’initiale représente Jimenez, le nom de naissance de la mère de Maria. Une initiale peut même représenter plus d’un seul nom&nbsp;: <span class="qchar">D</span> peut signifier ‘Dela Cruz’ lorsque le nom est écrit dans son intégralité.</p>
```

Comment:
```suggestion
    <p>Les Philippins écrivent également leur nom avec une initiale au milieu, mais celle‑ci représente le nom de naissance de la mère. Par exemple, pour <b class="highlight">Maria J. Go</b>, l’initiale représente Jimenez, le nom de naissance de la mère de Maria. Une initiale peut même représenter plus d’un seul nom&nbsp;: <span class="qchar">D</span> peut signifier «&nbsp;Dela Cruz&nbsp;» lorsque le nom est écrit dans son intégralité.</p>
```

(I know I used single quotes here and there, but come to think of it it's not very correct, typography-wise, in French. @arkhi I'll let you decide whether keeping single quotes everywhere or changeing everything to double French quotes.)

Reply:
I think I’ll let you answer first next time. :}

I was surprised about all those single quotes so I appreciate you’re bringing this up. I’ll change to double‑quotes.

Does anybody have any objection to using Unicode characters instead of HTML entities?

Reply:
Based on my [previous comment][1], I would replace those quotes by italic unless one has reasons not to.

[1]: https://github.com/w3c/i18n-drafts/pull/262#discussion_r484852942

Reply:
Yeah, why not, I'm ok with it. But maybe @r12a has something to say.

Reply:
@r12a: I tried to apply changes to the document in its entirety and went a bit mad. :}

I wonder if we should better **get the ball rolling; I can submit a different PR** with those changes and include more needed updates, like changing all `&nbsp;` to narrow non‑breakable spaces (`&#8239;`), etc.

---

If I understand properly the [usage of `<i>`](https://www.w3.org/International/questions/qa-b-and-i-tags.en), _idiomatic text_ would be used for most of the text that is not in the language of the article and is not used as the name of the person itself. That means a lot of the text could be updated  with:

```html
<i class="foreign" lang="[…]">[…]</i>
```

In the following example, only the first _appellidos_ is used as an idiomatic text, alhtough I understand `<i>` should wrap all occurrences. I would change `<span>` to `<i class="foreign" lang="es">` if that was just me, but I guess this was thought out before and I would prefer not to break anything (and learn about why).

```html
<i class="foreign" lang="es">apellidos</i>, sont Pérez Quiñones parce que les <span lang="es">apellidos</span> de son père sont Pérez Rodriguez et les <span lang="es">apellidos</span> […]
```

All text in a foreign language should also be marked with its `lang` attribute, I guess? Am I too extreme to think so? :)

Reply:
My 2cts: I always add `lang` attributes, for accessibility's sake.

Reply:
All apellidos should indeed be surrounded by `<i class="foreign" lang="en">...</i>`.  I have changed this in the english version.  Foreign language text should have lang attributes.